analysis/report_templates/default.html

```diff
@@ -259,7 +259,17 @@ <h5 class="center-align">Bug coverage sample statistics</h5>
                         <br>
 
                         <div class="row">
-                            <div class="col s6 offset-s3">
+                            <div class="col s6">
+                                <h5 class="center-align">Vargha-Delaney A measure</h4>
+                                    <img class="responsive-img materialboxed"
+                                         src="{{ benchmark.bug_vargha_delaney_plot }}">
+                                    The table summarizes the A12 values from the
```

Comment:
Would you mind adding a paragraph on A12 to the docs too:
https://google.github.io/fuzzbench/reference/report/
https://github.com/google/fuzzbench/blob/master/docs/reference/report.md
Could be later in a different PR too if you prefer

Reply:
Will do, also in a follow-up PR.